From 2f4228c1081fb73f237ae888d8162eb019d15337 Mon Sep 17 00:00:00 2001
From: Thomas Pugliese <thomas.pugliese@gmail.com>
Date: Sun, 6 Oct 2013 13:48:47 -0500
Patch-Mainline: v3.13
References: bnc#852581
Git-Commit: afc3cba5395a9f46de7cd50ba0e54a70b1d7b2b1
Subject: [PATCH 16/24] usb: wusbcore: fix string formatting warnings on 64-bit
 builds

This patch fixes compile warnings on 64-bit builds that were introduced
by the recent isoc changes.

Reported-by: kbuild test robot <fengguang.wu@intel.com>
Reported-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Thomas Pugliese <thomas.pugliese@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/usb/wusbcore/wa-xfer.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/wusbcore/wa-xfer.c
+++ b/drivers/usb/wusbcore/wa-xfer.c
@@ -471,7 +471,7 @@ static ssize_t __wa_xfer_setup_sizes(str
 		xfer->segs = DIV_ROUND_UP(urb->transfer_buffer_length,
 						xfer->seg_size);
 		if (xfer->segs > WA_SEGS_MAX) {
-			dev_err(dev, "BUG? oops, number of segments %d bigger than %d\n",
+			dev_err(dev, "BUG? oops, number of segments %zu bigger than %d\n",
 				(urb->transfer_buffer_length/xfer->seg_size),
 				WA_SEGS_MAX);
 			result = -EINVAL;
@@ -1921,7 +1921,7 @@ static void wa_process_iso_packet_status
 	dev_dbg(dev, "DTI: isoc packet status %d bytes at %p\n",
 		urb->actual_length, urb->transfer_buffer);
 	if (urb->actual_length != expected_size) {
-		dev_err(dev, "DTI Error: isoc packet status--bad urb length (%d bytes vs %zu needed)\n",
+		dev_err(dev, "DTI Error: isoc packet status--bad urb length (%d bytes vs %d needed)\n",
 			urb->actual_length, expected_size);
 		goto error_parse_buffer;
 	}
